presto-main/src/main/java/com/facebook/presto/operator/LookupSourceSupplier.java

```diff
@@ -21,4 +21,9 @@
     long getHashCollisions();
 
     double getExpectedHashCollisions();
+
+    /**
```

Comment:
Drop this. Or replace with
`checksum of this entity for heuristic checking equivalence of two instances`